objects/src/accounts/account_id.rs

```diff
@@ -1,7 +1,11 @@
 use super::{Account, AccountError, Digest, Felt, Hasher, StarkField, ToString, Vec, Word};
+use assembly::utils::to_hex;
 use core::fmt;
 use crypto::FieldElement;
 
+#[cfg(not(feature = "std"))]
+use alloc::string::String;
```

Comment:
We can probably change this to:
```
#[cfg(feature = "log")]
use crypto::utils::string::String;
```

Reply:
changed.

what is the benefit of those re-exports?